dataeng/jobs/AnalyticsExporter.groovy

```diff
@@ -0,0 +1,257 @@
+    }
+
+    multiscm{
+        git {
+            remote {
+                url('git@github.com:edx/edx-platform.git')
+                branch('$PLATFORM_BRANCH')
+                credentials('1')
+            }
+            extensions {
+                pruneBranches()
+                relativeTargetDirectory('edx-platform')
+            }
+        }
+        git {
+            remote {
+                url('git@github.com:edx/edx-analytics-exporter.git')
+                branch('$EXPORTER_BRANCH')
+                credentials('1')
+            }
+            extensions {
+                pruneBranches()
+                relativeTargetDirectory('analytics-exporter')
+            }
+        }
+        git {
+            remote {
+                url(commonConfigMap.get('SECURE_REPO_URL'))
+                branch('$SECURE_BRANCH')
+                credentials('1')
+            }
+            extensions {
+                pruneBranches()
+                relativeTargetDirectory('analytics-secure')
+            }
+        }
+        git {
+            remote {
+                url(commonConfigMap.get('BAKED_CONFIG_SECURE_REPO_URL'))
+                branch('*/master')
+                credentials('1')
+            }
+            extensions {
+                relativeTargetDirectory('config/baked-config-secure')
+            }
+        }
+    }
+
+    wrappers {
+        timestamps()
+    }
+
+    steps {
+        virtualenv {
+            nature("shell")
+            command(
+                readFileFromWorkspace("dataeng/resources/setup-platform-venv.sh")
+            )
+        }
+        virtualenv {
+            nature("shell")
+            command(
+                readFileFromWorkspace("dataeng/resources/setup-course-exporter.sh")
+            )
+        }
+    }
+}
+
+job ('analytics-exporter-slave') {
+
+    parameters {
+        stringParam('NOTIFICATION_EMAILS')
+        stringParam('MASTER_WORKSPACE')
+        stringParam('ORG_CONFIG_PATH')
+        stringParam('GPG_KEYS_PATH')
+        stringParam('DATE')
+        stringParam('CONFIG_PATH')
+        stringParam('OUTPUT_BUCKET')
+        stringParam('EXPORTER_VENV')
+        stringParam('ORG')
+        stringParam('PLATFORM_VENV')
+        stringParam('EXTRA_OPTIONS')
+    }
+
+    throttleConcurrentBuilds {
+        maxPerNode(4)
+        maxTotal(4)
+    }
+
+    concurrentBuild()
+
+    multiscm {
+        git {
+            remote {
+                url(commonConfigMap.get('BAKED_CONFIG_SECURE_REPO_URL'))
+                branch('*/master')
+                credentials('1')
+            }
+            extensions {
+                relativeTargetDirectory('config/baked-config-secure')
+            }
+        }
+    }
+
+    wrappers {
+        timestamps()
+        buildName('#${BUILD_NUMBER} ${ENV,var="ORG"}')
+    }
+
+    steps {
+        shell(readFileFromWorkspace("dataeng/resources/course-exporter-worker.sh"))
```

Comment:
Should analytics-exporter-slave call org-exporter-worker.sh, instead of course-exporter-worker.sh, or something like that?  The name course-exporter-worker.sh seems a better fit to be called by analytics-exporter-course.

Reply:
Updated the name of script.